testutil/feeburner/keeper/feeburner.go

```diff
@@ -0,0 +1,53 @@
+package keeper
+
+import (
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/store"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	typesparams "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/neutron-org/neutron/x/feeburner/keeper"
+	"github.com/neutron-org/neutron/x/feeburner/types"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/tendermint/libs/log"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	tmdb "github.com/tendermint/tm-db"
+)
+
+func FeeburnerKeeper(t testing.TB) (*keeper.Keeper, sdk.Context) {
```

Comment:
Is this a constructor? Maybe `NewFeeburnerKeeper()` then?

Reply:
That's actually automatically generated by ignite, and naming convention for this function is the same across testutils for all other modules.

Reply:
aaah okay